tests/data/cpu_template_helper/fingerprint_AMD_MILAN_4.14host.json

```diff
@@ -1,5 +1,5 @@
 {
-  "firecracker_version": "1.6.0-dev",
+  "firecracker_version": "1.7.0-dev",
```

Comment:
Do those matter? I don't know if the intent here is to match the current version or that this fingerprint was taken with Firecracker v1.6.0-dev @zulinx86

Reply:
Fixed following offline discussion.

Reply:
It seems the snapshot files do need to be updated https://buildkite.com/firecracker/firecracker-pr/builds/8062#018c6929-0d8a-46c1-b6e3-6dfadef17d3a

Reply:
Yeah that's in snapshots. That error is not related to these fingerprint files.

Reply:
> I don't know if the intent here is to match the current version or that this fingerprint was taken with Firecracker v1.6.0-dev.

The reason why a fingerprint has the field is to say which Firecracker version was used to take it, because the content can change between different Firecracker versions.